apps/web/public/static/locales/en/common.json

```diff
@@ -507,6 +507,7 @@
   "your_name": "Your name",
   "your_full_name": "Your full name",
   "no_name":"No name",
+  "enter_number_between_range": "Please enter a number between 1 and 20.",
```

Comment:
I think this should be a variable based on the maximum occurences set by the event type?

Reply:
okk

Reply:
Done

Reply:
> I think this should be a variable based on the maximum occurences set by the event type?

Done